Modules/Core/Common/include/itkMultiThreaderBase.h

```diff
@@ -213,7 +220,9 @@ class ITKCommon_EXPORT MultiThreaderBase : public Object
       UNKNOWN
     } ThreadExitCode;
   };
-  ITK_GCC_PRAGMA_DIAG_POP()
+// clang-format off
+ITK_GCC_PRAGMA_DIAG_POP()
```

Comment:
What can this macro nest with? Perhaps `// clang-format off` is not needed for this instance.